package.json

```diff
@@ -48,7 +56,16 @@
     "selectorator": "^4.0.1"
```

Comment:
The situation is better now than before, but I agree that `reselect` would be vastly superior because it only accepts functions, and is typed so that types can be inferred automatically.

That being said, changing the library (and thus the public API) is out of this PR's scope, which is only about translating the existing API.